.github/scripts/install_start_yugabyte.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) YugaByte, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied.  See the License for the specific language governing permissions and limitations
+# under the License.
+
+set -exo pipefail
+
+. /etc/os-release
+
+export ARCHITECTURE=`uname -m`
+export MOS=`uname|tr '[:upper:]' '[:lower:]'`
+
+if [[ $ARCHITECTURE != "x86_64" && $ARCHITECTURE != "aarch64" ]]; then
+  echo "Arch $ARCHITECTURE not yet supported"
+  exit 1
+fi
+
+if [[ $ARCHITECTURE == "aarch64" ]]; then
+  # For aarch64 yugabyte build we have el8 in build name
+  # e.g: yugabyte-2.15.1.0-b175-el8-aarch64.tar.gz & yugabyte-2.15.1.0-b175-linux-x86_64.tar.gz
+  export MOS="el8"
+fi
+
+if [[ "${ID_LIKE:-}" == "rhel fedora" ]]; then
```

Comment:
instead of `"rhel fedora"` we could use `*rhel*`

Reply:
updated the code